### DISCUSSION TITLE
DBA Dash GUI for macbook

### DISCUSSION THREAD

**Discussion by IndrajitMurgude** [trimble-oss/dba-dash#724] [Q&A]
do we have mac compatible version for DBA Dash GUI version

**Comment by DavidWiseman** [comment]
Hi, sorry there isn't a version compatible with mac.  I don't have any experience with macOS or porting apps to macOS.  I'm aware there are some options for running Windows apps on mac but I don't know if they will work with DBA Dash.  Another workaround might be to run the DBA Dash GUI from a Windows VM.
